internal/server/data/migrations.go

```diff
@@ -637,3 +619,19 @@ func deleteDuplicates(tx *gorm.DB, group string, model models.Modelable) error {
 	subQuery := tx.Select("min(id)").Group(group).Model(model)
 	return tx.Where("id NOT in (?)", subQuery).Delete(model).Error
 }
+
+// nolint:revive
+func addFieldsFor_0_14() *migrator.Migration {
+	return &migrator.Migration{
+		ID: "202207211828",
+		Migrate: func(tx *gorm.DB) error {
+			if err := tx.AutoMigrate(&models.Provider{}); err != nil {
+				return err
+			}
+			if err := tx.AutoMigrate(&models.Settings{}); err != nil {
+				return err
+			}
```

Comment:
TODO: I should replace these with sql `ALTER TABLE` statements if possible.

Reply:
Just a heads up, sqlite's support for alter table is really limited.